dropwizard-testing/src/test/java/io/dropwizard/testing/app/PersonResourceTest.java

```diff
@@ -21,6 +24,14 @@
  * Tests {@link ResourceTestRule}.
  */
 public class PersonResourceTest {
+    @Provider
```

Comment:
I believe the `@Provider` annotation is redundant, because we explicitly register this exception mapper.